mesonbuild/mdist.py

```diff
@@ -48,8 +48,7 @@ def create_hash(fname):
     m = hashlib.sha256()
     m.update(open(fname, 'rb').read())
     with open(hashname, 'w') as f:
-        f.write('{} {}\n'.format(m.hexdigest(), os.path.basename(fname)))
-    print(os.path.relpath(fname), m.hexdigest())
+        f.write('{}  {}\n'.format(m.hexdigest(), os.path.basename(fname)))
```

Comment:
Please leave a comment here explaining why there are two spaces here, or it's going to get changed again in the future.

Reply:
Good idea